### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by sindresorhus**
Enjoy!

### CODE REVIEW

**Comment by lukechilds** [comment]
Awesome! Ontsagwekkende! رائع! Выдатна! Страхотно! অসাধারণ! Increïble! Kahibulongan! Skvělý! Fantastisk! Genial! Φοβερός! ¡Increíble! Vinge! عالی! Mahtava! Impressionnant! Impresionante! અદ્ભુત! Nasara! बहुत बढ़िया! Fantasztikus! Հիանալի! Luar biasa! Egwu! Æðislegur! Eccezionale! מדהים! 驚くばかり！ Apik! გასაოცარია! Тамаша! ល្អមែនទែន! ಅದ್ಭುತ! 굉장해! Nuostabu! Satriecošs! Mahatalanjona! Ataahua! Прекрасно! ആകർഷണീയമായ! Гайхалтай! अप्रतिम! Biża &#39;! अति उत्तम! Geweldig! Rått! Zozizwitsa! ਸ਼ਾਨਦਾਰ! Niesamowite! Impressionante! Minunat! Потрясающие! නියමයි! Úžasné! Mbresëlënës! Авесоме! E hlollang! Heboh! Grymt bra! அற்புதம்! పరమాద్భుతం! Боварӣ! น่ากลัว! Kahanga-hanga! Korku veren! Чудово! خوفناک! Ajoyib! Tuyệt vời! Oniyi! 真棒！:tada:

File: app/locales/sv-SE/portfolio.json [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
-		"securityDescription": "<0>We recommend storing it offline. <1>Learn more security best practices</1></0>",
-		"securityTitle": "Important: please back up your seed phrase now!",
-		"seedPhrase": "Seed Phrase for Your Portfolio",
-		"seedPhraseNoMatch": "The seed phrase you entered is not the same as the generated one",
-		"success": "New portfolio added!",
-		"title": "Create New Portfolio"
+		"confirm": "Bekräfta",
+		"confirmPassword": "Bekräfta lösenord",
+		"confirmPasswordNoMatch": "Bekräftat lösenord matchar inte lösenord",
+		"confirmSeedPhrase": "Bekräfta din fröfras",
```
Comment (kevva) [inline app/locales/sv-SE/portfolio.json]: This sounds so funny translated. Maybe we should keep "seed phrase" in English?

File: app/locales/sv-SE/dashboard.json [hunk 2]
```diff
@@ -1,53 +1,53 @@
...
+		"recentActivity": "Senaste aktivitet",
+		"swapFailedTitle": "Ingen swap",
+		"swapTitle": "Swap genomförd",
+		"swapDescription": "{{quoteCurrency}} för {{baseCurrency}}"
 	},
 	"chart": {
-		"noData": "No data available",
-		"portfolioValue": "Portfolio Value",
-		"symbolChart": "{{symbol}} Chart"
+		"noData": "Inga data tillgängliga",
```
Comment (kevva) [inline app/locales/sv-SE/dashboard.json]: While correct, I feel like "Ingen data tillgänglig" is better. The above says more like "No datas available" in plural.

File: app/locales/sv-SE/chart.json [hunk 3]
```diff
@@ -1,3 +1,3 @@
 {
-	"noData": "No data available"
+	"noData": "Inga data tillgängliga"
```
Comment (kevva) [inline app/locales/sv-SE/chart.json]: Same as below.

File: app/locales/sv-SE/portfolio.json (hunk 1 above)
Comment (lukechilds) [reply app/locales/sv-SE/portfolio.json]: Yeah, Sindre mentioned the same thing when he translated to Norwegian. We should just use English for "seed phrase".

"Alice" and "Bob" should be in english too as they are technical terms not names in this context. But we should remove all references to Alice/Bob at some point to some more user friendly labels.

**Pull request merged**